src/components/Pagination.jsx

```diff
@@ -41,14 +43,15 @@ function Pagination({
           className="arrowButton left"
           // Do not remove the aria-label below, it is used for Hatchways automation.
           aria-label="Goto previous page"
-          onClick={onPrevious}
-          disabled={false} // change this line to disable a button.
+          onClick={(e)=>onPrevious(e)}
```

Comment:
The spacing of the code seems off. It can be consistent with the spacing of the starter code.